src/applications/mhv-medical-records/containers/LandingPage.jsx

```diff
@@ -191,6 +193,27 @@ const LandingPage = () => {
           </Link>
         </section>
       )}
+      {displayMedicalRecordsSettings && (
+        <section>
+          <h2 className="vads-u-margin-top--4 vads-u-margin-bottom--1">
+            Manage your medical records settings
+          </h2>
+          <p className="vads-u-margin-bottom--2">
+            Review and update your medical records sharing and notification
+            settings.
+          </p>
+          <Link
+            to={mhvUrl(
+              isAuthenticatedWithSSOe(fullState),
+              'electronic-record-sharing-options',
+            )}
+            className="vads-c-action-link--blue"
+            data-testid="settings-link"
+          >
```

Comment:
Just verifying that this link is supposed to point to MHV Classic, is that right?

Reply:
yes I got it from UCD. I imitated a similar link and changed the endpoint.